.github/workflows/main.yaml

```diff
@@ -28,6 +28,9 @@ jobs:
       - name: Install deps
         run: |
           npm install
+      - name: Prettier check 💅
+          run: |
+            npm run prettier-check
```

Comment:
Noen spesiell grunn til å gjøre det sånn, i stedet for `npx prettier --check`, eller blir det akkurat det samme?

Reply:
Det blir nok det samme (bortsett fra at . tar litt mer enn kun typescript). Kan endre til det du foreslår her!

Reply:
@JMLindseth fikk noen rare tslint-feil jeg ikke fant ut av så kjapt, og tslint er fjernet i en annen branch jeg har. Så fjerner github actionen her - legger den heller til igjen i nyebranchen :)

Reply:
Den er grei! Jeg har ingen preferanser på om det er et eget script eller om man bruker npx, bare lurte.